AirCasting/CoreData/MeasurementStreamStorage.swift

```diff
@@ -103,12 +103,8 @@ final class HiddenCoreDataMeasurementStreamStorage: MeasurementStreamStorageCont
         let sessionEntity = try context.existingSession(uuid: sessionUUID)
         let toDelete = sessionEntity.allStreams!.filter({ $0.gotDeleted })
         toDelete.forEach { object in
-            do {
-                toDelete.forEach { object in
-                    context.delete(object)
-                }
-            } catch {
-                Log.error("Error when deleting session")
+            toDelete.forEach { object in
+                context.delete(object)
             }
         }
```

Comment:
I changed it only to get rid of the warning that catch would never be reached because the call doesn't throw. I think it's from your ticket for deleting sessions

Reply:
But you are right, it does seem weird. You don't remember why was it done this way?

Reply:
Need to say that have no idea, but now I remember that it was for something so maybe we can leave it for now?

Reply:
Sure, I'll just add a warning 😉 😄